model/custom_pokemon_model.py

```diff
@@ -15,15 +17,21 @@ def __init__(self):
 
         self.numerical_fc = nn.Linear(7 * 12, EMBEDDING_DIM)  # 7 stats for each pokemon on both teams
 
+        self.lstm = nn.LSTM(
```

Comment:
Ah man, now I have to update the model diagram which took forever to draw 🫠

Reply:
Haha I can help with that if needed. We may end up altering it even more before we finish 👀